Dockerfile.py3

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && \
+  apt-get install -qy software-properties-common python3-software-properties && \
+  apt-get update && apt-get install -qy \
+    build-essential \
+    wget \
+    python3 \
+    r-base \
+    libffi-dev \
+    libssl-dev \
+    libjpeg-dev \
+    zlib1g-dev \
+    r-base \
+    libpython3-dev && \
+  apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://bootstrap.pypa.io/get-pip.py && python3 get-pip.py
+
+
+WORKDIR /girder_worker
+COPY setup.py /girder_worker/setup.py
+COPY requirements.txt /girder_worker/requirements.txt
+COPY requirements.in /girder_worker/requirements.in
+COPY README.rst /girder_worker/README.rst
+COPY examples /girder_worker/examples
+COPY scripts /girder_worker/scripts
+COPY girder_worker /girder_worker/girder_worker
+COPY docker-entrypoint.sh /girder_worker/docker-entrypoint.sh
+
+RUN pip3 install -e .
+
+RUN useradd -D --shell=/bin/bash && useradd -m worker
+
+# RUN /usr/local/bin/girder-worker-config set girder_worker tmp_root /tmp
+RUN chown -R worker:worker /girder_worker
+
+USER worker
+
+RUN cp /girder_worker/girder_worker/worker.dist.cfg /girder_worker/girder_worker/worker.local.cfg && \
+    sed -i girder_worker/worker.local.cfg \
+    -e '/^broker/ s/guest@localhost/%(RABBITMQ_USER)s:%(RABBITMQ_PASS)s@%(RABBITMQ_HOST)s/'
```

Comment:
Is there a way we could pass this as a command line argument somewhere instead of having to use advanced hackery to edit the config file?

Reply:
Yeah,  one of the things i'd like to do is rework the entire config mechanism in girder worker,  but for now this is just focused on replicating the python2 steps in a python3 Dockerfile.

Reply:
If there isn't a runtime invocation command to use, there's always the `girder-worker-config set` command that could be used here instead of `sed`.